docs/how-to-guides/propagating-updates.md

```diff
@@ -0,0 +1,64 @@
+# Propagating updates 
+
+This resource seeks to offer direction for those needing to provide updates to content, whether in a template for a theme, pattern, or a block over an entire site. 
+
+## How to approach managing updates
+
+**Establish early what content you expect to require updates**
+
+At a high level, it’s important to recognize that not every piece of content can be updated across the entire site and that the method of creation greatly impacts what’s possible. As a result, it’s critical to spend time ahead of creation determining what you expect to need updates and to put that content in the appropriate format. This will make a huge difference in terms of future maintenance. 
+
+**Embrace theme design at the block level**
+
+Block theme design requires a mindset shift from the previous approach of designing large sections of a theme and controlling them via updates. While a holistic view of a design is still important when creating a custom theme project, blocks require that themers approach design on a more atomic level. This means starting from the block itself, typically through theme.json customizations. The goal is that each individual "atom" (i.e., block) can be moved around, edited, deleted, and put back together without the entire design falling apart.
+
+The more that you approach design at the block level, the less need there is to propagate updates to things like patterns and templates across the entire site. If the atomic pieces are in place, their layout should not matter.
+
+## Overview of content types
+
+**Blocks**
+
+Dynamic blocks with the Save() method defining a default output is the recommended approach for blocks that you expect to change over time. The updates on any block will be automatically applied as the main HTML output will be always generated on render time by PHP code of the block. To get started on creating blocks and save time, [you can use the Create Block tool](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/).
+
+Generally speaking, this is the preferred approach due to the overhead involved with managing [block deprecations](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) and the requirement to manually update existing blocks otherwise. Keep in mind that the biggest downside with dynamic blocks comes from needing to manage markup in two places. Depending on your needs and comfortability, either approach can work. 
```

Comment:
Two notes here:
- It might sound like a critique of the block deprecations system. It's a valid concern that it's overhead and hard to maintain. However, we should keep emphasising that it's more about the nature of the block. If it depends on external data, then you need to use the dynamic block anyway. If the block is expected to change over time, then `render_callback` is a better choice as it gives better control. However, that flexibility comes at the cost of additional processing during rendering.
- I think that with both dynamic and static blocks you still need to manage markup in two places: `edit` + `save` vs `edit` + `render_callback`. The difference is that in the first case it's JavaScript only, but in the second case we mix JavaScript and PHP. As of today reusing all code between `edit` and `save` isn't possible as discussed in https://github.com/WordPress/gutenberg/issues/43201. However, there is a plan to mitigate that. We need to remember also that `edit` and `save` will always have differences, because they have different purposes.

Reply:
Do you mind editing this directly or suggesting how best to succinctly phrase? This is where @justintadlock and myself have less experience. 

What do you think of the following?

>How to manage block updates depends on the nature of the block itself. If it depends on external data, dynamic blocks with the Save() method defining a default output is the recommended approach. If the block is expected to change over time, then the `render_callback` is often a better choice as it provides more control. Keep in mind that that flexibility and controls comes at the cost of additional processing during rendering. Another option is to rely on managing updates with [block deprecations](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/). This will require you to manually update exist blocks.  Depending on your needs and comfortability, either approach can work. To get started on creating blocks and save time, [you can use the Create Block tool](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/).

Reply:
I would see it closer to this:

> How to manage block updates depends on the nature of the block itself. If the block depends on external data, then making it dynamic from start with the `render_callback` function is often a better choice as it provides more control. If the block's structure is expected to change over time, then starting with the static block that uses `save()` method defining a default output is the recommended approach. Over time, it's possible to go hybrid and include also the `render_callback` that can use the output from `save` as a fallback while processing an alternate output. Keep in mind that that flexibility and controls comes at the cost of additional processing during rendering. Another option is using static blocks that rely on managing updates with [block deprecations](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/). This will require you to manually update exist blocks.  Depending on your needs and comfortability, either approach can work. To get started on creating blocks and save time, [you can use the Create Block tool](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/).

Reply:
Updated :) Thank you.